views/SVGs/Eventos/Eventos.module.scss

```diff
@@ -0,0 +1,8 @@
+
+.cls-2 {
+  width: 35px;
+}
+
+.cls-2 path {
```

Comment:
please check: https://github.com/Azordev/did-admin-panel/pull/160/files#r963128551

I think you can consult this Mozilla guide: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute

In case not exist any attribute to use, you can [add inline styles to the path](https://www.w3schools.com/html/html_css.asp)

`style`

Reply:
So I checked and converted to class successfully.